proteinworkshop/datasets/pdb_dataset.py

```diff
@@ -130,21 +124,28 @@ class PDBDataModule(ProteinDataModule):
     def __init__(
         self,
         path: Optional[str] = None,
+        structure_dir: Optional[str] = None,
         pdb_dataset: Optional[PDBData] = None,
         transforms: Optional[Iterable[Callable]] = None,
         in_memory: bool = False,
         batch_size: int = 32,
         num_workers: int = 0,
         pin_memory: bool = False,
+        structure_format: str = "mmtf.gz",
```

Comment:
Can you update the relevant config file to ensure it has all args present.

Reply:
done @a-r-j